packages/contracts/src/core/plugin/PluginUUPSUpgradeable.sol

```diff
@@ -47,8 +47,8 @@ abstract contract PluginUUPSUpgradeable is
 
     /// @notice Returns the address of the implementation contract in the [proxy storage slot](https://eips.ethereum.org/EIPS/eip-1967) slot the [UUPS proxy](https://eips.ethereum.org/EIPS/eip-1822) is pointing to.
     /// @return implementation The address of the implementation contract.
-    function getImplementationAddress() public view returns (address implementation) {
-        implementation = _getImplementation();
+    function implementation() public view returns (address) {
+        return _getImplementation();
     }
```

Comment:
Shouldn't there be an `immutable` getter instead? or doesn't apply to the own plugin, itself?

Reply:
This is the implementation address stored in the `PluginUUPSUpgradeable` coming` from OZ's `UUPSUpgradeable.sol`, not from `PluginSetup`.